python/python/glide/async_commands/core.py

```diff
@@ -131,6 +131,23 @@ class UpdateOptions(Enum):
     GREATER_THAN = "GT"
 
 
+class Coordinate:
+    def __init__(self, longitude: float, latitude: float):
+        """
+            Represents a geographic position defined by longitude and latitude.
+
+        The exact limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the following:
+            - Valid longitudes are from -180 to 180 degrees.
+            - Valid latitudes are from -85.05112878 to 85.05112878 degrees.
+
+        Args:
+            longitude (float): The longitude coordinate.
+            latitude (float): The latitude coordinate.
+        """
+        self.longitude = longitude
```

Comment:
Do you want to check arguments are withing the range?

Reply:
I am not sure, since redis already provides us an error for that

Reply:
no need to test unless we crash in case of bad values - please check that we do not crash with bad values